### PR TITLE
 Add staking phase 1.1 info

### PR DESCRIPTION
**Pull request by bb111189**

### CODE REVIEW

File: docs/staking/phase1/staking-phase11-notice.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- Removal of user-defined ADT in `AssignStakeRewards` transition
+- Staking parameters change
+
+The migration to phase 1.1 will involve the following: 
+1. Freezing of phase 1.0 contracts
+2. Deployment of new proxy and ssnlist contracts
+3. Migration of states and funds from phase 1 to phase 1.1
+
+## Important note
+- The Contracts address will be change after the migration. Wallets and explorer will need to point to new contract addresses.
```
Comment (AmritKumar) [inline docs/staking/phase1/staking-phase11-notice.md]:
```suggestion
- The contract addresses will be changed after the migration. Wallets and explorers will need to point to the new contract addresses.
```

File: docs/staking/phase1/staking-phase11-notice.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+- No staking activities can be performed during the migration 
+- No action is needed for **delegators**
+- For **wallets, explore and node operator**, you will need to switch to the new contract addresses after the migration. All transition name and parameter in the contracts remain the same.
```
Comment (AmritKumar) [inline docs/staking/phase1/staking-phase11-notice.md]:
```suggestion
- For **wallets, explorers and node operators**, you will need to switch to the new contract addresses after the migration. All transition names and parameters in the contracts remain the same.
```

File: docs/staking/phase1/staking-phase11-notice.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+- Stake rewards will be retroactively be rewarded in full to all delegators and node operators for the period of migration
```
Comment (AmritKumar) [inline docs/staking/phase1/staking-phase11-notice.md]:
```suggestion
- Stake rewards will be retroactively rewarded in full to all delegators and node operators for the period of migration
```

File: docs/staking/phase1/staking-phase11-notice.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+- We will also update Zillion staking portal to point to Phase 1.1 contracts
+
+## Contracts information
```
Comment (AmritKumar) [inline docs/staking/phase1/staking-phase11-notice.md]:
```suggestion
## Information on contracts
```

File: docs/staking/phase1/staking-phase11-notice.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+For wallets and explorers, we will aim to let you know the phase 1.1 mainnet staking contract addresses as soon as we can. In the meanwhile, we have migrated the testnet staking contract
```
Comment (AmritKumar) [inline docs/staking/phase1/staking-phase11-notice.md]:
```suggestion
For wallets and explorers, we will aim to let you know the phase 1.1 mainnet staking contract addresses as soon as we can. In the meantime, we have migrated the testnet staking contract
```

File: docs/staking/phase1/staking-phase11-notice.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+from phase 1 to phase 1.1. Please refer to [this page](staking-general-information) for Phase 1.1 testnet contracts.
+
+## Changes to staking parameters (Mainnet)
+
+With mainnet `v8.0.0` upgrade, block time will be reduced. As such we will be making the following staking parameters changes
```
Comment (AmritKumar) [inline docs/staking/phase1/staking-phase11-notice.md]:
```suggestion
With mainnet `v8.0.0` upgrade, block time will be reduced. As such, we will be making the following staking parameter changes
```

File: docs/staking/phase1/staking-phase11-notice.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+For wallets and explorers, you may need to adjust your UI to reflect the changes in the parameters. 
+
+## Migration duration
+
+| | Duration | 
+|-------------- | ------------- | 
+| Starting Date | Tuesday 11th May 2021 04:45 t * |
+| Ending Date | Tuesday 18th May 2021 05:00* |
+*In the event of migration ending early than scheduled, we will resume the staking activities at the early possible time
```
Comment (AmritKumar) [inline docs/staking/phase1/staking-phase11-notice.md]:
```suggestion
*In the event of migration ending earlier than scheduled, we will resume the staking activities at the earliest possible time
```

**Review by AmritKumar — APPROVED** [review]

**Comment by teye** [comment]
https://github.com/Zilliqa/dev-portal/blob/feat/staking11/docs/staking/phase1/overview.md

Need to fix the link (docs/staking/phase1/general-information) to (docs/staking/phase1/staking-general-information)

**Comment by bb111189** [comment]
> https://github.com/Zilliqa/dev-portal/blob/feat/staking11/docs/staking/phase1/overview.md
> 
> Need to fix the link (docs/staking/phase1/general-information) to (docs/staking/phase1/staking-general-information)

nice catch. fixed

File: docs/staking/phase1/overview.md [hunk 8]
```diff
@@ -33,7 +33,7 @@ Please read our [Disclaimer](../staking-disclaimer) page carefully before partic
 
 The next sections in this phase 1 documentation are organized as follows:
 
-1. The [General Information](general-information) section contains useful details for anyone interested in the staking program
+1. The [General Information](docs/staking/phase1/staking-general-information) section contains useful details for anyone interested in the staking program
```
Comment (teye) [inline docs/staking/phase1/overview.md]: `The [General Information](staking-general-information)`

Comment (bb111189) [reply docs/staking/phase1/overview.md]: opps. Fixed

**Review by teye — APPROVED** [review]

**Review by renlulu — APPROVED** [review]

**Review by ansnunez — APPROVED** [review]

**Pull request merged**